app/src/main/java/com/example/cryptocurency/ui/asset_fragment/AssetFragment.kt

```diff
@@ -34,6 +162,17 @@ class AssetFragment : BaseFragment<FragmentAssetBinding>(FragmentAssetBinding::i
         }
     }
 
+    private fun updateAssetCoinData(coins: MutableList<Coin>) {
+        coins.map { coin ->
```

Comment:
dùng chính bản thân cái coin thì dùng map chi 
```suggestion
        coins.foreach { coin ->
```